src/pages/studyView/StudyViewPage.tsx

```diff
@@ -85,46 +150,36 @@ export default class StudyViewPage extends React.Component<{}, {}> {
 
     constructor(){
         super();
-
         this.store = new StudyViewPageStore();
 
-        const reaction1 = reaction(
-            () => this.store.filterTerm,
-            term => console.log("reaction 1:", term)
-        );
-
-    }
-
-    @observable tooltipState = true;
-
-    @action handleQueryChange(){
-
-        this.store.filterTerm=this.queryInput.value;
-
     }
+    
+    renderAttributeChart = (data:ClinicalAttribute, arrayIndex:number) =>
+	{
+		return (
+			<div className={styles.chart} key={arrayIndex}>
+                <div className={styles.header}>
+                    <span>{data.displayName}</span>
+                </div>
+                <div className="plot">
+                    <PieChart store={this.store} data={this.store.cinicalAttributeData.result[data.clinicalAttributeId]}/> 
```

Comment:
seem comment about store.  Try to pass presentational components the specific data they need instead of stores.